### ISSUE TITLE
compose(::Visual, ...)

### ISSUE THREAD

**Issue by kalmarek** [oscar-system/Polymake.jl#128]
doesn't work

but
```julia
P = rand_sphere(3,10)
Q = rand_sphere(3,10)

Common.compose(Polytope.visual(P).obj, Polytope.visual(Q).obj)
```
seems to work fine

**Comment by sebasguts** [comment]
It is the same problem as in #129, I think. I will fix this.

**Comment by kalmarek** [comment]
i don't think it's the same thing: `visual(P)` returns a `Visual` struct;

here is my one-liner:
```julia
Common.compose(vs::Vararg{Polymake.Visual, N}) where N = Common.compose([v.obj for v in vs]...)
```

**Comment by sebasguts** [comment]
Yeah, you can do it like this, too.

The thing is that `internal_call_{function/method}` does not know about the `Visual` type, so passing it directly to the polymake function does not work.

If you extract the object on the Julia level, it of course works. You can still not use the low-level interface then.

If you want to make it possible to pass the `Visual` type on all levels, one needs to do something similar to what I did in #131 for `BigInt` etc.

**Comment by kalmarek** [comment]
fixed by #133
